integration/cdi/src/main/java/org/apache/cxf/cdi/JAXRSCdiResourceExtension.java

```diff
@@ -473,7 +473,7 @@ private List< Object > loadServices(final BeanManager beanManager, Collection<Cl
     private List< Object > loadBeans(final BeanManager beanManager, Collection<Class<?>> limitedClasses,
                                      Collection<Bean<?>> beans) {
         // Use set to account for singletons and application scoped beans
```

Comment:
The comment now does not align anymore. You'd need to filter out duplicates by other means?

Reply:
It never had been right technically or at best it was not reliable with this impl will just drop it.